### PR TITLE
Add Docusaurus

### PR DESCRIPTION
**Pull request by Hongarc**
Docusaurus create a website with Markdown + React.js.

Feature: 
 - Document Versioning: Support users on all versions of your project. Document versioning helps you keep documentation in sync with project releases
 - Document Search: Make it easy for your community to find what they need in your documentation (Algolia DocSearch)
 - Translations: Localization comes pre-configured (Crowdin )

### CODE REVIEW

**Comment by sindresorhus** [comment]
Please follow the contribution guidelines.

**Comment by sindresorhus** [comment]
You are still not following. Please read it thoroughly. And the description could be improved.

**Comment by yangshun** [comment]
By submitting this pull request, I promise I have read the [contribution guidelines](https://github.com/sindresorhus/awesome-nodejs/blob/master/contributing.md) twice and ensured my submission follows it. I realize not doing so wastes the maintainers' time that they could have spent making the world better. 🖖

⬆⬆⬆⬆⬆⬆⬆⬆⬆⬆

GitHub: https://github.com/facebook/docusaurus
Website: https://www.docusaurus.io

Docusaurus is a performant documentation website generator that leverages on React and Markdown. It comes with translations and versioning features built-in.

File: readme.md [hunk 1]
```diff
@@ -359,6 +359,7 @@
 - [ESDoc](https://github.com/esdoc/esdoc) - Documentation generator targeting ES2015, attaching test code and measuring documentation coverage.
 - [Docco](https://github.com/jashkenas/docco) - Documentation generator which produces an HTML document that displays your comments intermingled with your code.
 - [JSDoc](https://github.com/jsdoc3/jsdoc) - API documentation generator similar to JavaDoc or PHPDoc.
+- [Docusaurus](https://github.com/facebook/docusaurus) - Documentation website with Markdown + React.js, easy to maintain
```
Comment (yangshun) [inline readme.md:362]: I think the description is better as "Docusaurus is a performant documentation website generator that leverages on React and Markdown. It comes with translations and versioning features built-in."

Comment (sindresorhus) [reply readme.md:362]: Almost perfect. I prefer to leave out marketing words like "performant", "simple", etc, as anyone could say that about their project.

Comment (Hongarc) [reply readme.md]: I create a commit with the description: `Documentation website generator that leverages on React and Markdown. It comes with translations and versioning features built-in.`

Thank you @yangshun and @sindresorhus

**Comment by Hongarc** [comment]
I updated the description. (removed `Docusaurus is a `)

**Pull request merged**